### PR TITLE
Fix doc build example failures

### PR DESCRIPTION
**Pull request by christopherlovell**
Fixes examples so that (a) during the docs build, if they fail, the build fails, and (b) that they are updated to emission models, as these were missed in the original switch over because of (a)

## Issue Type
- Bug

## Checklist
- [x] I have read the [CONTRIBUTING.md]() -->
- [x] I have added docstrings to all methods
- [x] I have added sufficient comments to all lines
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no pep8 errors
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -50,13 +50,11 @@
...
-    # Modules for which function/class level galleries are created. In
-    # this case sphinx_gallery and numpy in a tuple of strings.
+    # Modules for which function/class level galleries are created
     "doc_module": ("synthesizer"),
-    #     # Regexes to match objects to exclude from implicit backreferences.
-    #     # The default option is an empty set, i.e. exclude nothing.
-    #     # To exclude everything, use: '.*'
-    #     # 'exclude_implicit_doc': {r'pyplot\.show'},
+    "abort_on_example_error": True,
+    # "show_memory": True,
```
Comment (WillJRoper) [inline docs/source/conf.py:56]: Any reason to leave these here commented?

Comment (WillJRoper) [reply docs/source/conf.py:56]: If show memory shows the memory usage in the logs that sounds useful!

**Comment by WillJRoper** [comment]
Looks good to me, what's left to do on this?

**Comment by christopherlovell** [comment]
Still a lot, none of the cosmo examples work presently. Please hold :)

File: docs/source/conf.py (hunk 1 above)
Comment (christopherlovell) [reply docs/source/conf.py:56]: I thought about it but it's expensive. I think okay to leave here for users to comment out when testing locally?

**Review by WillJRoper — APPROVED** [review]

**Pull request merged**